pkg/apic/provisioning/propertybuilder.go

```diff
@@ -36,6 +36,7 @@ type propertyDefinition struct {
 	Minimum            *float64                      `json:"minimum,omitempty"`  // We use a pointer to differentiate the "blank value" from a chosen 0 min value
 	Maximum            *float64                      `json:"maximum,omitempty"`  // We use a pointer to differentiate the "blank value" from a chosen 0 max value
 	IsEncrypted        bool                          `json:"x-axway-encrypted,omitempty"`
+	TextArea           string                        `json:"x-axway-widget,omitempty"`
```

Comment:
May be this can be named as Widget and the SetAsTextArea sets the widget as "textarea"

Reply:
yeah, will do, since the widget can be of type widget